WorkWithDatabaseLibrary/MSSQLLocalDBRepository.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Data.SqlClient;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace WorkWithDatabaseLibrary
+{
+    public class MSSQLLocalDBRepository : IRepository<Record>
+    {
+        private static readonly string dataSource = @"(LocalDB)\MSSQLLocalDB";
+        private static readonly string databaseFilename = @$"{Directory.GetCurrentDirectory()}\WorkDatabase.mdf";
```

Comment:
I would rather pass this as a parameter to a constructor to make the class more flexible